### DISCUSSION TITLE
Running obfuscated code with mono on linux

### DISCUSSION THREAD

**Discussion by zblojko** [mkaring/ConfuserEx#476] [Q&A]
Hi,

is it possible to protect exe file and run it using mono on linux (Raspberry Pi)?

I am getting error:

Unhandled Exception:
System.NotImplementedException: The method or operation is not implemented.
  at <Module>.Main (System.String[] args) [0x0003b] in <20edaff446f141d687bd8ed8bc3c4b3d>:0
[ERROR] FATAL UNHANDLED EXCEPTION: System.NotImplementedException: The method or operation is not implemented.
  at <Module>.Main (System.String[] args) [0x0003b] in <20edaff446f141d687bd8ed8bc3c4b3d>:0

Thank you

Darko

**Comment by mkaring** [comment]
Some protections should work, but not all by a long shot. It depends on your protection configuration if that works or not,

I guess "Anti Debug", "Anti Dump","AntiTamper" and the compressor packer are likely causing issues. "Invalid Metadata" _will_ break the compatibility with mono.
